devito/core/autotuning.py

```diff
@@ -93,7 +93,9 @@ def autotune(operator, arguments, tunable):
         operator.cfunction(*list(at_arguments.values()))
         elapsed = sum(operator.profiler.timings.values())
         timings[tuple(bs.items())] = elapsed
-        info_at("<%s>: %f" % (','.join('%d' % i for i in bs.values()), elapsed))
+        iter_time = [s.extent(finish=at_arguments['time_size']) for s in sequential][0]
```

Comment:
I think a better way is to check as soon as possible that ``sequential`` contains only one loop and then extract it straight away

Reply:
ie right after line 33